TCAT/Views/SummaryView.swift

```diff
@@ -10,72 +10,49 @@ import UIKit
 
 class SummaryView: UIView {
 
-    /// The puller tab used to indicate dragability
-    private let tab = UIView()
-
-    /// Three times the height of the tab view (spacing + tabHeight + spacing)
-    private let tabInsetHeight: CGFloat = 12
-
-    /// The usable height of the summaryView
-    var safeAreaHeight: CGFloat {
-        return frame.size.height - tabInsetHeight
-    }
-
-    /// The y-coordinate center of the safe area
-    private var safeAreaCenterY: CGFloat {
-        return tabInsetHeight + safeAreaHeight / 2
-    }
-
-    /// The primary summary label
-    private let mainLabel = UILabel()
-
-    private let labelsContainerView = UIView()
-
     private var iconView: UIView!
-
-    /// The live indicator
+    private let labelsContainerView = UIView()
     private let liveIndicator = LiveIndicator(size: .small, color: .clear)
-
-    /// The secondary label (Trip Duration)
+    private let mainLabel = UILabel()
     private let secondaryLabel = UILabel()
+    private let tab = UIView()
 
-    /// Whether route icons have been set or not
-    private var didSetRoutes: Bool = false
-
-    /// Constant for label padding
-    private let textLabelPadding: CGFloat = 16
-    /// Identifier for bus route icons
-    private let iconTag: Int = 14850
+    private let tabSize = CGSize(width: 32, height: 4)
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
     }
 
     init(route: Route) {
         // View Initialization
-        let height: CGFloat = 80 + tabInsetHeight
-        super.init(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: height))
+        super.init(frame: .zero)
+
+        /*
+         This value ends up getting overwritten by constraints, but for some reason
```

Comment:
Probably should be marked with a TODO or linked with an issue? You could also make the issue, and link the issue # in the comment itself!